pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -444,6 +446,29 @@ type EmailConfig struct {
 	TLSConfig *monitoringv1.SafeTLSConfig `json:"tlsConfig,omitempty"`
 }
 
+// VictorOpsConfig configures notifications via VictorOps.
+// See https://prometheus.io/docs/alerting/latest/configuration/#victorops_config
+type VictorOpsConfig struct {
```

Comment:
Don't we miss `CustomFields` here? This should be `[]KeyValue`.

Reply:
I don't see CustomFields is alertmanager configuration
https://prometheus.io/docs/alerting/latest/configuration/#victorops_config

if it is planned in future alertmanager release I believe it should be added later. But anyways if you want it to be added I'll add

Reply:
I should have added the link to the code in the first place :)
https://github.com/prometheus/alertmanager/blob/a7f9fdadbecbb7e692d2cd8d3334e3d6de1602e1/config/notifiers.go#L483

The documentation is out-of-date unfortunately.

Reply:
fixed in 348657e6bdd3d62f75a53bccd26686576ba5155d.